Games/types/Mafia/roles/cards/SubtractVoteWeight.js

```diff
@@ -0,0 +1,15 @@
+const Card = require("../../Card");
+
+module.exports = class SubtractVoteWeight extends Card {
+  constructor(role) {
+    super(role);
+
+    this.listeners = {
+      meeting: function (meeting) {
+        if (meeting.members[this.player.id]) {
+          meeting.members[this.player.id].voteWeight--;

```

Comment:
can you confirm again about negative vote weight? my main issue is people are going to get confused like "why does the vote not add up right" 

and it's function is the same as whistleblowing self + target